app/dashboard/_components/UploadThingModal.tsx

```diff
@@ -0,0 +1,69 @@
+'use client';
+
+import { setAppSetting } from '~/actions/appSettings';
+import { UploadThingTokenForm } from '~/app/(blobs)/(setup)/_components/UploadThingTokenForm';
+import Link from '~/components/Link';
+import {
+  Dialog,
+  DialogContent,
+  DialogDescription,
+  DialogHeader,
+  DialogTitle,
+} from '~/components/ui/dialog';
+import Paragraph from '~/components/ui/typography/Paragraph';
+
+function UploadThingModal() {
+  return (
+    <Dialog open={true}>
+      <DialogContent aria-describedby={undefined} className="sm:max-w-[500px]">
```

Comment:
We shouldn't ever really override the styling of such a fundamental component like this. If the width is wrong here, it should (in principle) be wrong everywhere, and so should be updated in the component itself. 

Also, if you _do_ need to make a one off adjustment, you shouldn't use arbitrary values for the same reason (prefer updating the breakpoints if those values aren't correct).